sound/soc/sof/intel/hda.h

```diff
@@ -376,11 +340,18 @@ struct sof_intel_hda_dev {
 	const struct sof_intel_dsp_desc *desc;
 
 	/*trace */
-	struct sof_intel_hda_stream *dtrace_stream;
+	struct hdac_ext_stream *dtrace_stream;
 
 	int irq;
 };
 
+#define SOF_STREAM_SD_OFFSET(s) (0x20 * ((s)->index) + SOF_HDA_ADSP_LOADER_BASE)
+
+#ifndef __SOUND_HDAUDIO_EXT_H
+#define stream_to_hdac_ext_stream(s) \
```

Comment:
Don't redefine this, just create a new macro

Reply:
OK, will remove this.